### PR TITLE
improve signal to noise ratio

### PR DESCRIPTION
**Pull request by krschultz**
filter out repos with < 5 stars & sort by star count rather than watchers

### CODE REVIEW

**Comment by emaloney** [comment]
+1

**Comment by haveahennessy** [comment]
:+1:

**Comment by rcaloras** [comment]
+1!

File: javascripts/main.js [hunk 1]
```diff
@@ -71,6 +72,10 @@ function addRepos(repos) {
   stats.removeClass("hidden");
 }
 
+function starFilter(repo) {
+  return repo.stargazers_count >= 5;
```
Comment (Seanstoppable) [inline javascripts/main.js]: Doesn't this effectively limit exposure of new projects?

Comment (krschultz) [inline javascripts/main.js]: Yes. If you want to have your project on the projects page, you need to get a few people to star it. If you ever need people to do that, drop into the open source chat, we can bump it up.

**Pull request merged**